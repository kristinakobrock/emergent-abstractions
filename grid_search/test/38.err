  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 18.30it/s] 10%|#########3                                                                                   | 5/50 [00:00<00:01, 22.52it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 23.86it/s] 22%|####################2                                                                       | 11/50 [00:00<00:01, 24.48it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 25.76it/s] 34%|###############################2                                                            | 17/50 [00:00<00:01, 22.64it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 23.43it/s] 46%|##########################################3                                                 | 23/50 [00:00<00:01, 23.97it/s] 52%|###############################################8                                            | 26/50 [00:01<00:01, 22.34it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 25.58it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 26.57it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 29.98it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 29.75it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 29.71it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 26.68it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.58it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 31%|############################6                                                                | 4/13 [00:00<00:00, 34.18it/s] 62%|#########################################################2                                   | 8/13 [00:00<00:00, 25.57it/s] 85%|#############################################################################8              | 11/13 [00:00<00:00, 25.59it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.82it/s]
