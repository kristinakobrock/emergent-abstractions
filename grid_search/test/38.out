# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7435608506202698, "acc": 0.5706833600997925, "length": 3.042733669281006, "mode": "train", "epoch": 1}
{"loss": 0.6883448958396912, "acc": 0.5494140982627869, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6727610230445862, "acc": 0.6055183410644531, "length": 1.2233809232711792, "mode": "train", "epoch": 2}
{"loss": 0.6625295877456665, "acc": 0.627148449420929, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6635008454322815, "acc": 0.6212157011032104, "length": 1.1634986400604248, "mode": "train", "epoch": 3}
{"loss": 0.6568074822425842, "acc": 0.6298828125, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6602553129196167, "acc": 0.6232550740242004, "length": 1.125725269317627, "mode": "train", "epoch": 4}
{"loss": 0.6554883718490601, "acc": 0.6296875476837158, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6599840521812439, "acc": 0.6232019066810608, "length": 1.0980886220932007, "mode": "train", "epoch": 5}
{"loss": 0.6553704142570496, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6597520112991333, "acc": 0.6234965920448303, "length": 1.0834836959838867, "mode": "train", "epoch": 6}
{"loss": 0.6553690433502197, "acc": 0.6273437142372131, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6594107747077942, "acc": 0.624049186706543, "length": 1.0813628435134888, "mode": "train", "epoch": 7}
{"loss": 0.6569814085960388, "acc": 0.627734363079071, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6565913558006287, "acc": 0.6227959990501404, "length": 1.1843935251235962, "mode": "train", "epoch": 8}
{"loss": 0.6561649441719055, "acc": 0.6279296875, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6411485075950623, "acc": 0.6348024606704712, "length": 1.747436285018921, "mode": "train", "epoch": 9}
{"loss": 0.6261379718780518, "acc": 0.6510986089706421, "length": 2.3125, "mode": "test", "epoch": 9}
{"loss": 0.6172833442687988, "acc": 0.6605240106582642, "length": 2.0656635761260986, "mode": "train", "epoch": 10}
{"loss": 0.6337101459503174, "acc": 0.6423766613006592, "length": 1.64453125, "mode": "test", "epoch": 10}
{"loss": 0.6144937872886658, "acc": 0.6635100841522217, "length": 1.8253164291381836, "mode": "train", "epoch": 11}
{"loss": 0.6395410299301147, "acc": 0.658618152141571, "length": 2.23046875, "mode": "test", "epoch": 11}
{"loss": 0.6062726378440857, "acc": 0.6695799827575684, "length": 1.977504849433899, "mode": "train", "epoch": 12}
{"loss": 0.6378365755081177, "acc": 0.6442016363143921, "length": 2.06640625, "mode": "test", "epoch": 12}
{"loss": 0.5986682176589966, "acc": 0.6743995547294617, "length": 2.2021331787109375, "mode": "train", "epoch": 13}
{"loss": 0.6321802139282227, "acc": 0.6680663824081421, "length": 2.3828125, "mode": "test", "epoch": 13}
{"loss": 0.5897642970085144, "acc": 0.6832346320152283, "length": 2.407794952392578, "mode": "train", "epoch": 14}
{"loss": 0.6036566495895386, "acc": 0.6768646836280823, "length": 2.5234375, "mode": "test", "epoch": 14}
{"loss": 0.5737451314926147, "acc": 0.7004112005233765, "length": 2.5229411125183105, "mode": "train", "epoch": 15}
{"loss": 0.5928756594657898, "acc": 0.689300537109375, "length": 2.74609375, "mode": "test", "epoch": 15}
{"loss": 0.55101078748703, "acc": 0.7185074090957642, "length": 2.6205759048461914, "mode": "train", "epoch": 16}
{"loss": 0.5903083086013794, "acc": 0.7063812017440796, "length": 2.5859375, "mode": "test", "epoch": 16}
{"loss": 0.5375486016273499, "acc": 0.7304544448852539, "length": 2.6808273792266846, "mode": "train", "epoch": 17}
{"loss": 0.5641435384750366, "acc": 0.7119750380516052, "length": 2.75390625, "mode": "test", "epoch": 17}
{"loss": 0.5235301852226257, "acc": 0.7310779094696045, "length": 2.8041372299194336, "mode": "train", "epoch": 18}
{"loss": 0.5572147965431213, "acc": 0.7243744134902954, "length": 3.17578125, "mode": "test", "epoch": 18}
{"loss": 0.5123298764228821, "acc": 0.7494823932647705, "length": 2.8971407413482666, "mode": "train", "epoch": 19}
{"loss": 0.55977863073349, "acc": 0.7273437976837158, "length": 2.65625, "mode": "test", "epoch": 19}
{"loss": 0.4971064627170563, "acc": 0.7580589056015015, "length": 2.796962022781372, "mode": "train", "epoch": 20}
{"loss": 0.5329907536506653, "acc": 0.7369048595428467, "length": 2.89453125, "mode": "test", "epoch": 20}
{"loss": 0.4836430251598358, "acc": 0.7642603516578674, "length": 2.9590249061584473, "mode": "train", "epoch": 21}
{"loss": 0.5353444814682007, "acc": 0.7333008050918579, "length": 2.81640625, "mode": "test", "epoch": 21}
{"loss": 0.476712167263031, "acc": 0.7721888422966003, "length": 3.1921801567077637, "mode": "train", "epoch": 22}
{"loss": 0.5522712469100952, "acc": 0.7171416878700256, "length": 3.38671875, "mode": "test", "epoch": 22}
{"loss": 0.46617814898490906, "acc": 0.7796728610992432, "length": 3.486309289932251, "mode": "train", "epoch": 23}
{"loss": 0.5091406106948853, "acc": 0.7537841796875, "length": 3.88671875, "mode": "test", "epoch": 23}
{"loss": 0.4512133002281189, "acc": 0.7875359058380127, "length": 3.80446720123291, "mode": "train", "epoch": 24}
{"loss": 0.5140919089317322, "acc": 0.7671875357627869, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.4355461597442627, "acc": 0.7975941300392151, "length": 3.931530475616455, "mode": "train", "epoch": 25}
{"loss": 0.4865108132362366, "acc": 0.7654296159744263, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.41832786798477173, "acc": 0.8073916435241699, "length": 3.9787750244140625, "mode": "train", "epoch": 26}
{"loss": 0.4660438001155853, "acc": 0.7835937142372131, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.41062018275260925, "acc": 0.8136144876480103, "length": 3.98421311378479, "mode": "train", "epoch": 27}
{"loss": 0.47517067193984985, "acc": 0.7777343392372131, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.4039757549762726, "acc": 0.8142535090446472, "length": 3.9905147552490234, "mode": "train", "epoch": 28}
{"loss": 0.46435457468032837, "acc": 0.7849608063697815, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.3914455473423004, "acc": 0.821344256401062, "length": 3.9924516677856445, "mode": "train", "epoch": 29}
{"loss": 0.44849181175231934, "acc": 0.7978515028953552, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.3824310302734375, "acc": 0.8299732804298401, "length": 3.9962761402130127, "mode": "train", "epoch": 30}
{"loss": 0.46923041343688965, "acc": 0.8019532561302185, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.3734051287174225, "acc": 0.8324346542358398, "length": 3.996527671813965, "mode": "train", "epoch": 31}
{"loss": 0.4564135670661926, "acc": 0.8070311546325684, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.3567589223384857, "acc": 0.8403353691101074, "length": 3.996204137802124, "mode": "train", "epoch": 32}
{"loss": 0.4314751625061035, "acc": 0.8111328482627869, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3496130108833313, "acc": 0.8462916016578674, "length": 3.991913318634033, "mode": "train", "epoch": 33}
{"loss": 0.46080487966537476, "acc": 0.7947266101837158, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.35527902841567993, "acc": 0.8477457165718079, "length": 3.9985814094543457, "mode": "train", "epoch": 34}
{"loss": 0.4485418200492859, "acc": 0.8089843392372131, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.36110812425613403, "acc": 0.843876838684082, "length": 3.997514009475708, "mode": "train", "epoch": 35}
{"loss": 0.4306151270866394, "acc": 0.8154296875, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.34931138157844543, "acc": 0.8496944904327393, "length": 3.9993956089019775, "mode": "train", "epoch": 36}
{"loss": 0.4280385971069336, "acc": 0.8242188096046448, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.3378349840641022, "acc": 0.8551784157752991, "length": 3.9982476234436035, "mode": "train", "epoch": 37}
{"loss": 0.41522353887557983, "acc": 0.8263672590255737, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3266697824001312, "acc": 0.8611170649528503, "length": 3.999769687652588, "mode": "train", "epoch": 38}
{"loss": 0.4069192409515381, "acc": 0.8251952528953552, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3188572824001312, "acc": 0.8613322973251343, "length": 3.998594045639038, "mode": "train", "epoch": 39}
{"loss": 0.3893200159072876, "acc": 0.8318358659744263, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.31042614579200745, "acc": 0.8706541657447815, "length": 3.99961256980896, "mode": "train", "epoch": 40}
{"loss": 0.4245108664035797, "acc": 0.8324218392372131, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.3057177662849426, "acc": 0.8746187090873718, "length": 3.999103307723999, "mode": "train", "epoch": 41}
{"loss": 0.4217342734336853, "acc": 0.8296875357627869, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.31352609395980835, "acc": 0.8712395429611206, "length": 3.9996581077575684, "mode": "train", "epoch": 42}
{"loss": 0.4142610430717468, "acc": 0.8408203125, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.3125115931034088, "acc": 0.8717145323753357, "length": 3.99971079826355, "mode": "train", "epoch": 43}
{"loss": 0.39182084798812866, "acc": 0.8404297232627869, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.2929646968841553, "acc": 0.8804351091384888, "length": 3.9996960163116455, "mode": "train", "epoch": 44}
{"loss": 0.38236308097839355, "acc": 0.8507813215255737, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.3047592341899872, "acc": 0.878140389919281, "length": 3.9991207122802734, "mode": "train", "epoch": 45}
{"loss": 0.41434741020202637, "acc": 0.8423828482627869, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.30079907178878784, "acc": 0.8826079964637756, "length": 3.9999399185180664, "mode": "train", "epoch": 46}
{"loss": 0.4290927052497864, "acc": 0.8326172232627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.31523987650871277, "acc": 0.8736122250556946, "length": 3.997654914855957, "mode": "train", "epoch": 47}
{"loss": 0.39795351028442383, "acc": 0.8505858778953552, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.29034119844436646, "acc": 0.8841326832771301, "length": 3.9999711513519287, "mode": "train", "epoch": 48}
{"loss": 0.3846311569213867, "acc": 0.8466796278953552, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.28237754106521606, "acc": 0.886590301990509, "length": 3.999943971633911, "mode": "train", "epoch": 49}
{"loss": 0.3612428903579712, "acc": 0.8603515028953552, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.2713342010974884, "acc": 0.8922711610794067, "length": 3.9999916553497314, "mode": "train", "epoch": 50}
{"loss": 0.37489742040634155, "acc": 0.8546875715255737, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.3167976438999176, "acc": 0.8704870343208313, "length": 3.9998819828033447, "mode": "train", "epoch": 51}
{"loss": 0.38889849185943604, "acc": 0.8412109017372131, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.2870205044746399, "acc": 0.8826402425765991, "length": 3.999436140060425, "mode": "train", "epoch": 52}
{"loss": 0.3711931109428406, "acc": 0.8566406965255737, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.27835118770599365, "acc": 0.8879873156547546, "length": 3.999810218811035, "mode": "train", "epoch": 53}
{"loss": 0.4073249101638794, "acc": 0.8541015982627869, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.2764691710472107, "acc": 0.8907797932624817, "length": 3.9993624687194824, "mode": "train", "epoch": 54}
{"loss": 0.36186403036117554, "acc": 0.8582030534744263, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.27126818895339966, "acc": 0.8941999077796936, "length": 3.9999120235443115, "mode": "train", "epoch": 55}
{"loss": 0.38519376516342163, "acc": 0.8585938215255737, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.2645343244075775, "acc": 0.8966872096061707, "length": 3.999408483505249, "mode": "train", "epoch": 56}
{"loss": 0.3607443571090698, "acc": 0.8658203482627869, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.2552313804626465, "acc": 0.8985911011695862, "length": 3.9999301433563232, "mode": "train", "epoch": 57}
{"loss": 0.37058156728744507, "acc": 0.8705077767372131, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.2507966458797455, "acc": 0.9028674364089966, "length": 3.9962189197540283, "mode": "train", "epoch": 58}
{"loss": 0.34494727849960327, "acc": 0.8779296875, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.250216543674469, "acc": 0.9028096795082092, "length": 3.9999852180480957, "mode": "train", "epoch": 59}
{"loss": 0.3580286502838135, "acc": 0.8736328482627869, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.24998220801353455, "acc": 0.9061475396156311, "length": 3.9999988079071045, "mode": "train", "epoch": 60}
{"loss": 0.37543004751205444, "acc": 0.8677735328674316, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7328333854675293
